### PR TITLE
Tag CategoricalArrays.jl v0.1.1

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaData/CategoricalArrays.jl](https://github.com/JuliaData/CategoricalArrays.jl)
Release: [v0.1.1](https://github.com/JuliaData/CategoricalArrays.jl/releases/tag/v0.1.1)
Travis: [Travis Build Status](https://travis-ci.org/JuliaData/CategoricalArrays.jl/builds/200734707)
Diff: [vs v0.1.0](https://github.com/JuliaData/CategoricalArrays.jl/compare/267b1025e4dd737b9cf54343c3f2803a8317d9b6...948d56256faf5367652803741b1cab2c29cd914d)
`requires` vs v0.1.0: 
```diff
--- 0.1.0/requires
+++ 0.1.1/requires
@@ -1,3 +1,3 @@
 julia 0.5.0-
-Compat 0.8.1
+Compat 0.9.5
 NullableArrays
```
cc: @nalimilan

### CODE REVIEW

File: CategoricalArrays/versions/0.1.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5.0-
```
Comment (tkelman) [inline CategoricalArrays/versions/0.1.1/requires]: do you still need to support prereleases?

Comment (nalimilan) [inline CategoricalArrays/versions/0.1.1/requires]: How fast! I don't think I ever cared about prereleases, but it there any reason not to support them?

Comment (tkelman) [inline CategoricalArrays/versions/0.1.1/requires]: this is essentially saying you support the first commit after release-0.4 branched

Comment (nalimilan) [inline CategoricalArrays/versions/0.1.1/requires]: Yes, ideally I would check which 0.5 prerelease fixed the crashes we experienced on 0.4.x. But it's not worth it, so I've removed the `-`.

**Pull request merged**